spa/src/components/Chat.vue

```diff
@@ -87,9 +87,10 @@
         </ul>
         <ul v-if="activePanel === 'gestures'">
           <li
-            v-for="(gesture, key) in this.$store.data.user.avatar.gestures"
+            v-for="(gesture, key) in this.$store.methods.getGestures()"
```

Comment:
If we set the user/gestures correctly in a function like `AppStore#setUsers`, do we need to get them back out via function? We access the store everywhere else by directly indexing into its data property, is that good? If accessing the store's data property remains reactive, then that's the way to go.

Reply:
reverted